### ISSUE TITLE
Workspace standard.options not work

### ISSUE THREAD

**Issue by cncolder** [standard/vscode-standard#23]
- [x] StandardJS version? 10.0.2
- [x] VSCode version? 1.11.2

I put config both in `package.json` and `.vscode/settings.json` then type `standard`. It skip `client` folder. But IDE still show error in `client/*.js`

My workspace setting like this:

```json
// ./.vscode/settings.json
{
    "standard.options": {
        "ignore": [
            "client",
            "node_modules"
        ]
    }
}
```

**Comment by ghost** [comment]
Same problem cannot ignore jasmine spec files.
```json
  "standard.options": {
      "ignore": [
          "node_modules/**",
          "src/**/*.spec.js"
      ]
  }

**Comment by chenxsan** [comment]
This bug is caused by [lintText](https://github.com/chenxsan/vscode-standardjs/blob/master/eslint-server/src/server.ts#L660), it ignores all the `ignore` setting. We will consider migrating to `lintFiles` method in future version to fix it.

**Comment by chenxsan** [comment]
@cncolder @daniel-leroux Just published 1.1.5 to fix it, please upgrade it.
